logicrunner/builtin/contract/migrationadmin/migrationadmin.go

```diff
@@ -0,0 +1,121 @@
+/*
+ *
+ *  Copyright  2019. Insolar Technologies GmbH
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package migrationadmin
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/logicrunner/builtin/foundation"
+)
+
+// MigrationAdmin manage and change status for  migration daemon.
+type MigrationAdmin struct {
+	foundation.BaseContract
+	MigrationDaemon      foundation.StableMap
+	MigrationAdminMember insolar.Reference
+}
+
+const (
+	StatusActive     = "ACTIVE"
```

Comment:
No need in string statuses. Use bool

Reply:
disagree

Reply:
Use bool or iota at least if you need more then 2 enums in future. 
There is no need in readable enums.

Reply:
for the future

Reply:
use int named constants

Reply:
We cant use any other types except strings in our StableMap for now